### PR TITLE
[CM_BE-12] Adding name and YOB and deleting firstName lastName and DOB

### PR DESCRIPTION
**Pull request by Apoorva-GA**

### CODE REVIEW

File: src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -356,4 +356,34 @@
...
+        </dropColumn>
+        <dropColumn tableName="patient">
+            <column name="date_of_birth" type="DATE">
+                <constraints nullable="true" />
+            </column>
+        </dropColumn>
+        <addColumn tableName="patient">
+            <column name="name" type="VARCHAR(100)">
+                <constraints unique="false"/>
+            </column>
```
Comment (ganesanarun) [inline src/main/resources/liquibase.xml]: varchar 100 is enough? firstname alone had 100 :)

Comment (Apoorva-GA) [reply src/main/resources/liquibase.xml]: Thought so, let me know If you want to keep it more.

Comment (ganesanarun) [reply src/main/resources/liquibase.xml]: https://www.liquibase.org/documentation/changes/rename_column.html won't work?

Comment (Apoorva-GA) [reply src/main/resources/liquibase.xml]: Will work, but do we want to rename anything here?

Comment (ganesanarun) [reply src/main/resources/liquibase.xml]: firstname to name, that way we will at least have firstname in the name column no?

Comment (Apoorva-GA) [reply src/main/resources/liquibase.xml]: Sure if we want to keep that data we can do that.

Comment (ganesanarun) [reply src/main/resources/liquibase.xml]: Thanks
We need some name, otherwise existing users will be ` ` Blank users :)

**Review by ganesanarun — APPROVED** [review]

**Pull request merged**